### PR TITLE
Resolved a user created password verification issue that could not exit

### PR DESCRIPTION
**Pull request by huzunhao**
1.The configure normal user password interface allows the user to exit without any configuration.
2.When normal user password is configured, choose not to require the password, and do not configure the password, allow the user to modify the user name can normally exit.

### CODE REVIEW

**Comment by poncovka** [comment]
@M4rtinK, can you look at this PR, please?

**Comment by huzunhao** [comment]
> Hello @huzunhao! Thanks for opening this PR. We checked the lines you've touched for [PEP 8](https://www.python.org/dev/peps/pep-0008) issues, and found:
> 
> * In the file [`pyanaconda/ui/gui/spokes/user.py`](https://github.com/rhinstaller/anaconda/blob/3574f24e74fa6fe76fc5ac9dcd335d2b4bd7fe61/pyanaconda/ui/gui/spokes/user.py):
> 
> > [Line 509:1](https://github.com/rhinstaller/anaconda/blob/3574f24e74fa6fe76fc5ac9dcd335d2b4bd7fe61/pyanaconda/ui/gui/spokes/user.py#L509): [W293](https://duckduckgo.com/?q=pep8%20W293) blank line contains whitespace

ok,I will modify for it

**Comment by VladimirSlavik** [comment]
@huzunhao I think it complains about the tabulator on the "empty" line.

**Comment by huzunhao** [comment]
> @huzunhao I think it complains about the tabulator on the "empty" line.

ok，I have deleted the blank line.

**Comment by jkonecny12** [comment]
Jenkins, it's ok to test.

File: pyanaconda/ui/gui/spokes/user.py [hunk 1]
```diff
@@ -501,11 +505,11 @@ def password_required_toggled(self, togglebutton=None, data=None):
         password_is_required = togglebutton.get_active()
         self.password_entry.set_sensitive(password_is_required)
         self.password_confirmation_entry.set_sensitive(password_is_required)
-
+        self._password_is_required = password_is_required
```
Comment (jkonecny12) [inline pyanaconda/ui/gui/spokes/user.py:508]: Please remove these trailing white spaces.

Comment (huzunhao) [reply pyanaconda/ui/gui/spokes/user.py]: ok，I have remove these trailing white spaces.

Comment (huzunhao) [reply pyanaconda/ui/gui/spokes/user.py]: Any other questions？

**Comment by M4rtinK** [comment]
jenkins, test this please

**Comment by huzunhao** [comment]
> Looks good to me, thanks for the fix! :)
> 
> Please squash the three commits and then I think we can merge this. :)
Now I squash the three submissions and resubmit them in a new request
resolved a user created password verification issue that could not exit #2640

I will close the request, please merge #2640 for me

**Pull request closed without merging**